be/src/exec/pipeline/hdfs_chunk_source.cpp

```diff
@@ -318,7 +318,7 @@ Status HdfsChunkSource::buffer_next_batch_chunks_blocking(size_t batch_size, boo
 
     for (size_t i = 0; i < batch_size && !can_finish; ++i) {
         ChunkPtr chunk;
-        _status = _read_chunk_from_storage(_runtime_state, &chunk);
```

Comment:
this code should not been try catched now, it may use the expr framework and some code of storage layer

Reply:
I will remove this.